src/controllers/legacy/tenant.js

```diff
@@ -350,7 +350,7 @@ const approveSignUpRequest = async (ctx, next) => {
       return;
     }
 
-    const tx = await usersService.createUserAccount({ username, pubKey: userProfile.signUpPubKey, role });
+    const tx = await userService.createUserAccount({ username, pubKey: userProfile.signUpPubKey, role });
```

Comment:
Shouldn't this method fail ? You commented the createUserAccount method in the controller